### ISSUE TITLE
[Long term issue] Testing series four's ability to kill things other than malaria

### ISSUE THREAD

**Issue by MFernflower** [OpenSourceMalaria/OSM_To_Do_List#482]
I have mentioned this a few times before but I believe it would be best to leave this as a long term standing issue: Can the S4 drugs kill other pathogens besides Plasmodium falciparum? (Also, do these drugs work on vivax?)

I would be really interested in seeing if these drugs could kill TB (Wellcome trust will do this pro bono) and perhaps pathogenic free-living amoebæ like Nægleria Fowleri (Easily grown on coli plates) and Balamuthia mandrillaris (requires a special lab) 

I understand this is low on the priority list but I would love to hear peoples opinion on testing against these organisms!

**Comment by holeung** [comment]
That would be an interesting experiment. If S4 work by targeting ATPase4, then they are more likely to be effective in organisms closely related to plasmodium, with high sequence conservation of ATPase4. A quick search shows toxoplasma and cryptosporidium as the common pathogens with the most closely related ATPase4. TB, as mycobacteria, is not closely related.

**Comment by MFernflower** [comment]
Could you do a quick check to see about Nægleria Fowleri?

**Comment by MFernflower** [comment]
@holeung @mattodd I assume that COADD would be able to do the bacteriological testing pro bono? 

The issue is going to be getting a lab to test a few of these drugs against Toxoplasma and Nægleria respectively

**Comment by mattodd** [comment]
Thanks for raising this @MFernflower . The short answer is YES, we ought to be testing these compounds against other pathogens, all the time, by default. This was one of the rationales for ensuring we had good stocks of the Frontrunner set (#400). The bigger picture is that we'd like to have a mini "OSM-Box" of compounds that people could request, unencumbered by IP constraints, inspired by MMV's work in this area.

So what's the constraint? As usual, it's Time.

If people can identify assays to which we can submit compounds, and which involve a) a decent scientific rationale (since we are always resource-limited) and b) principles of open data , then we can ship things. What people need to do is contact the people who run those assays and see whether some OSM compounds can be accepted. And how they should be delivered, and how many compounds can be examined, etc. Building relationships like this takes a little time, to make sure everything is clear.

In fact we've already done a little bit of this. The data aren't public yet purely because of other things in OSM taking precedence, and I was wondering about the best way to make the data public since we have been evaluating a few compounds against pathogens that are beyond malaria, so where do the data go? Any thoughts on this, everyone? I'd been wondering about writing up a step-by-step on how to perform a single screen of known compounds and publish it as a mini-paper/preprint/dataset from where the data could most easily feed into pubchem. @cdsouthan 

The data I have sitting on my hard drive waiting to go public are screens of several compounds (including some OSM's) against _H. Contortus_ in the lab of Robin Gasser in Melbourne ([assay](http://www.ncbi.nlm.nih.gov.ezp.lib.unimelb.edu.au/pubmed/26365227)) (gave no hits, sadly) and a CO-ADD screen that identified OSM-S-293 as a confirmed hit vs _C. Albicans_ (inactive against other things). I need to share the data properly, so we can perhaps a) think how best to exploit the hit, and b) think what to target next. This would need *considerable community input* clearly, since the team here in Sydney are quite swamped just with the malaria work and only have limited bandwidth at the moment for stepping beyond that - as you can see from the delay in sharing the very nice Albicans hit... Comments and advice welcome, particularly re 

a) best methods for datasharing these individual non-malaria assays
b) high-value screens and how to get them done
c) mechanisms (beyond my lab) of ensuring we have stocks of good OSM compounds. Chris S - we've spoken before about this. Perhaps one for company philanthropy or schools-based crowdsourcing @alintheopen where compounds could be maintained at e.g. [Compounds Australia](https://www.griffith.edu.au/science-aviation/compounds-australia), ready for shipping.

**Comment by MFernflower** [comment]
"and a CO-ADD screen that identified OSM-S-293 as a confirmed hit vs C. Albicans (inactive against other things" I'm currently learning mycology and this is an interesting thing to be because if it's effective against an ascomycete I would be curious to see if it works against a basidiomycete (Cryptococcus neoformans) and a fungus-like pathogen (Nægleria fowleri). This is is a long term goal and will not be completed in the near future but I think this is important planning that needs to be done @mattodd 

I might be able to talk with the leader of the mycology club as he has some contacts but there is no guarantee

**Comment by holeung** [comment]
Shared sequence identity with Naegleria gluberi (Fowleri genomic data does not seem to be available yet) is 32%. For Candida albicans, it's 30%. ~30% is pretty low so I am surprised about the hit vs. C. albicans. For Haemonchus contortus, it's also 30%. Attached are the sequence homology analysis with BLAST against eukaryotes.

[NCBI Blast_Protein Sequence.pdf](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/756230/NCBI.Blast_Protein.Sequence.pdf)

**Comment by MFernflower** [comment]
35% for Cryptococcus!

**Comment by holeung** [comment]
Sequence identity with toxoplasma is 55% and cryptosporidium is 45%. I don't think there is strong interest in targeting these organisms as they generally are associated with mild disease except for immunocompromised patients. There is strong interest for developing antifungals vs C. albicans and other pathogenic fungi. My knowledge is limited to human diseases. It would be good to have someone look over the list to see what pathogens are important for animal disease.

**Comment by MFernflower** [comment]
Aphanomyces invadans seems like a good target for animal disease

**Comment by MFernflower** [comment]
Sorry. I clicked the wrong button on my phone

**Comment by mattodd** [comment]
Ok, please let me jump in here. If people are interested in particular screens can people please please to provide a little rationale (e.g. a lit reference to any existing screens vs that organism, or some clear indication that screens for new compounds are urgently needed) AND a location of a person who is known to be running the assay right now. The people who contributed to the construction of the Pathogen Box might be a good starting point?

**Comment by drc007** [comment]
I presume other OSM compounds were tested in the CO-ADD screen? If so and they are inactive I'd have thought it unlikely that they are acting via the same molecular mechanism ATPase4?
For data sharing the simplest method would be to create another Google doc spreadsheet with a column for each assay.

**Comment by cdsouthan** [comment]
I beg to differ. I would take up @mattodd s point on limited bandwidth and suggest "we" (i.e. the OSM enterprise) stay focussed on malaria, AWAK there is still  loads more to be done to get anywhere near a clinical candidate and  its not certain that Series 4 will deliver such.  NOBA there still seems to be a chronic shortage of reciprocal lead x-screening and/or mechanistic deconvolution even between the major malarial strains.  I am sanguine about the box approach since none of my antimalarial recomendations made it in https://cdsouthan.blogspot.se/2014/06/getting-into-box-with-some-recent.html

**Comment by mattodd** [comment]
Sure @drc007 the sharing of the data as a Google sheet is trivial (thanks to conversations we've all had on this subject) and can be accompanied by descriptions of assay methods from those people performing them. But where do these go, for maximal effect?

We can post them to an OSM lab book. We could post a report to Figshare, or Sydney Uni's repository. We could try to draft a Minimally Publishable Unit for a journal. What's best, in terms of piping the data into the most widely-employed databases, so that the results will be found most easily? What's the best method here that scales, so that others (interested in those other pathogens) can do the same kind of thing autonomously?

A new journal, The Cross-Screening Digest, anyone?

**Comment by cdsouthan** [comment]
"most widely-employed databases, so that the results will be found most easily?"  = PubChem BioAssay 
https://www.ncbi.nlm.nih.gov/pubmed/27899599  "PubChem BioAssay: 2017 update"

**Comment by mattodd** [comment]
Excellent, @cdsouthan thanks, that's great. With all the meta info there (i.e. authorship, brief backstory) I think this may be what we're looking for. [Example here](https://pubchem.ncbi.nlm.nih.gov/bioassay/1510#section=Top). It's selfishly useful for academics to be able to cite such things as evidence of work, and such a page is sufficiently permanent and citable to count, while also providing ease of discovery.

Do people agree? Anyone have any experience at a manual submission of this type?

**Comment by drc007** [comment]
I've sent an email to Ruth Neale at CO-ADD to see what they have done, no point in re-inventing the wheel.

**Comment by MFernflower** [comment]
@drc007 Great idea... Would it also be possible to ask if they (COADD) do screening against Cryptococcus neoformans and Nægleria fowleri?

**Comment by drc007** [comment]
@MFernflower I don't think so. Full details here. http://www.co-add.org/content/free-screening

**Comment by MFernflower** [comment]
@drc007 @mattodd Cryptococcus neoformans is done for free at COADD!!! Could we send a few compounds there way sometime? As for Nægleria fowleri.. We are stuffed unless somebody wants to do the testing for free... I think there is a charity just for Nægleria here in the USA (http://www.kylelewisamoebaawareness.org/) so they might be able to help??

**Comment by cdsouthan** [comment]
![capture](https://cloud.githubusercontent.com/assets/3713939/22708455/91443e72-ed76-11e6-92c3-1f5be80d6653.JPG)

**Comment by MFernflower** [comment]
That's pretty poor @cdsouthan


Im wondering if I should close this issue beause we cant find a testing lab
to do these screens? @mattodd

**Comment by drc007** [comment]
@MFernflower This issue has only been open 5 days! I'd leave for a little longer to allow time for infrequent visitors to comment.

**Comment by cdsouthan** [comment]
Agreed, these open discussions are useful  so no rush to "close"  As a summary perhaps 
1) active engagement with cross-screening outside malaria would not currently be considered a priority for  OSM resources
2) we obviously welcome utilisation of our results (including synthetic protocols) by others outside malaria
3) some issues touched on above are perennial and can be extended under other to-dos (e.g. intra-malarial compound sharing and data surfacing in PubChem)

**Comment by mattodd** [comment]
Let me summarise where I think we'd be most effective.

1) OSM's day-to-day is malaria.
2) A core feature of the project is the open source aspect, which must involve full protocols for everything, via the lab notebooks. This enables anyone to make all the molecules for themselves.
3) Nevertheless, it can be advantageous/barrier-lowering for there to be a central stock of the most promising antimalarials. We've tried to take this approach with the Frontrunners. In large part this makes internal OSM sense since we often come to realise that further evaluation of promising compounds is necessary. We can't centrally maintain stocks of everything, but an OSM-Box is relatively low effort if it additionally encourages:
4) ... members of the scientific community identifying gaps where OSM-Box compounds have not been evaluated for some activity/property. In such a case we could make it clear that members of the community are encouraged to identify possible applications of the compounds and request that samples be sent for evaluation somewhere.

If we maintained an OSM-Box at Compounds Australia, the shipping out would be relatively simple and cost effective.

If this seems reasonable (please chip in), we could include some How-To text on the wiki somewhere detailing all of this.

We could then publish a short note somewhere describing this approach and advocating that other labs maintain similar Boxes of the most promising compounds, and advocating for the publishing of the outputs into Pubchem along the lines of the discussion above.

**Comment by cdsouthan** [comment]
An OSM-Box sounds great.  Extending it beyond the front-runners with a few low-activity controls would be good, as well as including  other selected non-OSM leads (which the MMV boxes didn't really cover).  I  could assist with ensuring these were all mapped into PubChem

**Comment by MFernflower** [comment]
Will a box of the anti-fungal hits be stocked as well? OSM-Mycobox has a nice ring to it!

**Comment by JZuegg** [comment]
CO-ADD screened the compounds against Cryptococcus neoformans (H99; ATCC 208821) , so if we confirmed only activity against C.albicans, the activity against C.neoformans must have been below the threshold in the primary screen (80% inhibition @ 32 ug/mL). However, CO-ADD won't venture into parasite screen any time soon, so we can't help you there. We would like to screen molds though, but haven't managed to get the assay against Aspergillus fumigatus up and running... 
Let us know if confirmation (MIC) against C.neoformans is useful for your publication efforts.

For publication of data, CO-ADD can submit data directly to ChEMBL (strategic partner of CO-ADD) to be included in their next release. The option there are either with a publication as a reference, by issuing a DOI as a reference, or under the CO-ADD banner (not defined yet). So let me know, as we are happy to prep the data for ChEMBL import and sort out the issuing of a DOI (either by ChEMBL or here at UQ). The DOI would point to your set of data at ChEMBL.

An additional option I mentioned to an email to Chris is CDD (https://www.collaborativedrug.com/) where you can setup your own OSM collaborative site. I  ran into the CEO of CDD a few days ago, and he's very keen to get more open/public projects into CDD (like they have done with TB Alliance)... Let me know I can get you hooked up. 

And yes, let us know when you have a OSM-box or mycobox ready as we are happy to run them through our screen, especially if you have them at CpOz, which we use as storage and for plating as well.
 (j.zuegg@uq.edu.au; info@co-add.org)

**Comment by MFernflower** [comment]
@JZuegg Random question.. I'm wondering if you could figure out a screen against the fungus-like flagellate known as Nægleria fowleri or by it's common name "The brain eating ameoba"

**Comment by JZuegg** [comment]
In principle. Quick search found Naegleria fowleri from ATCC and a few published protocols to test it in a broth system. The devil lays usual in the details, especially scaling it to 96 or 384 plates. Any development and validation would take considerable time. I'll check with our microbiologist.

**Comment by cdsouthan** [comment]
@JZuegg  Thanks and JFTR I'm on the SAB of CDD and  so am aware of the powerful cross-platform integration and standardisation it offers.

**Comment by drc007** [comment]
http://www.cambridgemedchemconsulting.com/news/index_files/292de120ff03dbaddd9a14cb9224df39-262.html
WHO publishes list of bacteria for which new antibiotics are urgently needed

**Comment by MFernflower** [comment]
@drc007 Mr.Swain, Do you know if there is a list of fungi too? I do not think these drugs will kill bacteria as the proposed transporter they hit is only found on eukaryotes

**Comment by Jake-s16** [comment]
Don't some other antimalarials also kill _Babesia_ species? Maybe another group to look at.

**Comment by MFernflower** [comment]
@Jake-s16 Toxoplasma as well.. But the real ticket is showing that these drugs can kill fungi and fungus like organisms

**Comment by drc007** [comment]
I suspect that any activity against fungi will be via a different mechanism

**Comment by MFernflower** [comment]
@drc007 Uncertain - Fungi have very similar ATPases on their membranes - see @holeung 's comparison https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/756230/NCBI.Blast_Protein.Sequence.pdf

**Comment by MFernflower** [comment]
In a perfect world we would be able to throw 4 of our most potent TP's at Nægleria, Cryptococcus and Blasto and watch the carnage but time is a huge limitation

**Comment by MFernflower** [comment]
@mattodd Would you consider sending two active S4 and two inactive S4 drugs to COADD for testing against C. neoformans provided they can return the results within two months of them receiving the compounds?

**Comment by JZuegg** [comment]
Dear Todd,

Just to let you know, we are closing CO-ADD screening in the second half of December, for maintenance of assay/strains and calibration of equipment, as well as holiday over the Xmas period. We’ll restart screening in the second week of Jan-2018.

For any ‘fast’ results we would require the compounds with us in the next weeks.
Happy to run small batches as MIC right away.

Cheers
Johannes

Dr. Johannes Zuegg
Community for Open Antimicrobial Drug Discovery
Institute for Molecular Bioscience | The University of Queensland | Brisbane 4072 | Australia
T +61 (0)7 3346 2994 | E j.zuegg@uq.edu.au<mailto:j.zuegg@uq.edu.au> | Skype j.zuegg.uq | W www.co-add.org<http://www.co-add.org/>

[Co-Add_Logo_Web_Baseline]<http://www.co-add.org/>

**Comment by MFernflower** [comment]
@david1597 Do you have two active s4's and two inactive s4's to send off to @JZuegg  within the next week or so? Paging Dr.Todd!
